### DISCUSSION TITLE
Custom Tool - Einschränken anhand PVP-Rolle

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#101] [Q&A]
Hallo Jürgen!

Gibt es die Möglichkeit ein Custom Tool anhand einer PVP-Rolle sichtbar/unsichtbar zu schalten?

Es wurde eine Karte konfiguriert, die im User Access zwei PVP-Rollen konfiguriert hat: imap_view und imap_edit.
Außerdem wurde in der custom.js ein Custom Tool konfiguriert, welches einfach auf eine URL zeigt.  
Das Tool soll allerdings nur für die User mit der PVP-Rolle imap_edit angezeigt werden. 
Alle User mit der Rolle imap_view sollen in der Werkzeug-Leiste nichts von dem Tool sehen.
Über die custom.js schaffe ich es nicht das Tool anhand der PVP-Rolle sichtbar/unsichtbar zu schalten. 

Ist so eine Konfig möglich? Kann ich das in der custom.js konfigurieren oder muss das an einer anderen Stelle (cms?, AppBuilder - WebGIS-Redirect?) konfiguriert werden. 

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
eher nicht
